theme/material/vaadin-radio-group-styles.ts

```diff
@@ -0,0 +1,4 @@
+import { registerStyles } from '@vaadin/vaadin-themable-mixin/register-styles.js';
+import { radioGroupStyles } from './vaadin-radio-group-css';
+
+registerStyles('vaadin-radio-group', radioGroupStyles, { moduleId: 'lumo-radio-group' });
```

Comment:
`moduleId` should be updated to become `material`

Reply:
Done.